packages/static/kairos-overlay-files/files/system/oem/00_datasource.yaml

```diff
@@ -2,30 +2,30 @@ name: "Datasource handling"
 stages:
   rootfs.before:
     - &datasource
-      if: '[ ! -f /oem/userdata ] && ([ ! -f /run/cos/uki_boot_mode ] || grep -q "kairos.pull_datasources" /proc/cmdline )'
+      if: '[ ! -e /oem/userdata ] && ([ ! -f /run/cos/uki_boot_mode ] || grep -q "kairos.pull_datasources" /proc/cmdline )'
       name: "Pull data from provider"
       datasource:
         providers: ["cdrom", "gcp", "openstack", "aws", "azure", "hetzner", "packet", "vultr", "digitalocean", "metaldata", "vmware", "config-drive"]
-        path: "/oem"
-    - if: '[ ! -f /oem/userdata ]'
+        path: "/oem/userdata"
```

Comment:
you sure this is correct?

Reply:
no this is not, it will use this as the basepath and add userdata at the end if Im reading this correctly: https://github.com/mudler/yip/blob/master/pkg/plugins/datasource.go#L246

Reply:
yup this is wanted. I want to create files in `/oem/userdata` (directory)

Reply:
Current:
```
/oem/userdata /oem/userdata.yaml
```

wanted:

```
/oem/userdata/userdata /oem/userdata/userdata.yaml

```

So I can point `/oem/userdata` as a cloud init path